src/chesster.js

```diff
@@ -119,6 +119,36 @@ function leagueDMResponse(patterns, responseName) {
 leagueResponse(['captain guidelines'], 'formatCaptainGuidelinesResponse');
 leagueDMResponse(['captains', 'captain list'], 'formatCaptainsResponse');
 
+/* game nomination */
+chesster.hears({
+    middleware: [ slack.requiresLeague ],
+    patterns: [ 'nomination' ],
+    messageTypes: [ 'direct_message' ]
+},
+function(bot, message) {
+    if (!message.league) {
```

Comment:
I don't think this is necessary if you use slack.requiresLeague, but it doesn't hurt either.